components/brave_vpn/brave_vpn_service.cc

```diff
@@ -868,6 +870,11 @@ void BraveVpnService::OnCreateSupportTicket(
           << "\nresponse_code=" << status;
   std::move(callback).Run(success, body);
 }
+void BraveVpnService::OnSuspend() {
+  needs_connect_ = is_connected();
```

Comment:
Can you add some comments how just setting `needs_connect_` can solve this issue?

Reply:
done

Reply:
Thanks!